os-eap-launch/added/launch/datasource-common.sh

```diff
@@ -514,9 +514,6 @@ function inject_datasource() {
       checker="org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker"
       sorter="org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLExceptionSorter"
       ;;
-    "MONGODB")
-      continue
-      ;;
```

Comment:
Is there any issue related to this?

Reply:
No, its just a leftover, I can create one, but it should have been dropped with the PR that removed mongo originally.

Reply:
That's fine as is.